package.json

```diff
@@ -26,47 +24,59 @@
     "network",
     "browser"
   ],
+  "main": "dist/umd.js",
+  "module": "dist/esm.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist",
+    "LICENSE*"
+  ],
   "scripts": {
-    "test": "mocha --exit --require @babel/register",
-    "test-cov": "nyc --reporter=lcov mocha --exit --require @babel/register",
-    "build": "rollup --config rollup.config.js",
-    "watch": "rollup --watch --config rollup.config.js",
-    "watch-dev": "npm run watch-dev",
+    "build": "npm run build:types && npm run build:code && npm run build:docs",
+    "build:code": "rollup -c rollup.config.js",
+    "build:docs": "typedoc ./src",
+    "build:types": "tsc -p tsconfig.types.json",
+    "clean": "rimraf '{docs,dist}/*'",
+    "fix": "npm run lint -- --fix",
     "lint": "eslint '{src,test}/**/*.{m,}{j,t}s{x,}'",
-    "clean": "rimraf 'dist/*'",
-    "postinstall": "opencollective postinstall"
+    "postinstall": "opencollective postinstall",
+    "test": "BABEL_ENV=test mocha -r ./babel.mocha.js 'test/**/*.test.{j,t}s'",
+    "test-cov": "BABEL_ENV=test nyc --reporter=lcov mocha -r ./babel.mocha.js 'test/**/*.test.{j,t}s'",
+    "type-check": "tsc --noemit",
+    "watch": "rollup --watch --config rollup.config.js",
+    "watch-dev": "npm run watch-dev"
   },
   "dependencies": {
-    "vis-util": "^1.0.2"
+    "vis-util": "^1.1.0"
```

Comment:
There is not [vis-util@1.1.0 yet](https://www.npmjs.com/package/vis-util) :-)

Reply:
There is my pull request with 1.1.0.

Reply:
I need the TypeScript files from there for this to work.

Reply:
[vis-util@1.1.0](https://www.npmjs.com/package/vis-util) is now released